rero_ils/modules/acq_accounts/serializers/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Acquisition accounts serialization."""
+
+from invenio_records_rest.serializers.response import record_responsify, \
+    search_responsify
+
+from rero_ils.modules.acq_accounts.serializers.csv import \
+    AcqAccountCSVSerializer
+from rero_ils.modules.acq_accounts.serializers.json import \
+    AcqAccountJSONSerializer
+from rero_ils.modules.serializers import RecordSchemaJSONV1
+
+json_acq_account = AcqAccountJSONSerializer(RecordSchemaJSONV1)
```

Comment:
according to serialization refactoring, could you use `_json` and `_csv` name here : theses attributes should be private and never used outside of this file ; only "*_search" and "*_response" should be exposed (and maybe it will save enough spaces to write "*_search" on one line).

Use `__all__` to expose only "*_search" and "*_response". (I don't think other resource is used anywhere else)

Reply:
done